qubesadmin/tools/qvm_start.py

```diff
@@ -147,6 +147,10 @@ def main(args=None, app=None):
     for domain in args.domains:
         if args.skip_if_running and domain.is_running():
             continue
+        elif domain.is_running():
+            exit_code = 1
+            parser.print_error("domain is already running")
```

Comment:
This message should include domain name, because you may specify multiple of them. I'll add after merging.